src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -65,7 +71,8 @@ public static GivrUser getUserByIdFromDatastore(String userId) {
       throw new IllegalArgumentException("More than one user with the userId was found.");
     }
 
-    GivrUser user = new GivrUser(userId, isMaintainer);
+
+    GivrUser user = new GivrUser(userId, isMaintainer, isLoggedIn, "" /* URL will be set in AuthenticateServlet */);
```

Comment:
maybe reword the comment to mention you don't need the url if you are already logged in.